core/include/mm/core_mmu.h

```diff
@@ -295,6 +295,7 @@ extern const unsigned long core_mmu_tee_load_pa;
 
 void core_init_mmu_map(unsigned long seed, struct core_mmu_config *cfg);
 void core_init_mmu_regs(struct core_mmu_config *cfg);
+void core_mmu_save_mem_map(void);
```

Comment:
Maybee add a short description: `Copy static memory map from boot_mem to heap when CFG_BOOT_MEM is enabled`